scripts/clean-shrinkwrap.js

```diff
@@ -15,27 +15,42 @@
 
 'use strict';
 
+const _ = require('lodash');
+const fs = require('fs');
 const path = require('path');
-const packageJSON = require('../package.json');
-const spawn = require('child_process').spawn;
-const shrinkwrapIgnore = packageJSON.shrinkwrapIgnore;
 
-console.log('Removing:', shrinkwrapIgnore.join(', '));
+const NPM_SHRINKWRAP_FILE_PATH = path.join(__dirname, '..', 'npm-shrinkwrap.json');
+const shrinkwrapFile = require(NPM_SHRINKWRAP_FILE_PATH);
+const shrinkwrapIgnore = require('../package.json').shrinkwrapIgnore;
```

Comment:
Hmmm, but we could still use just `const NPM_SHRINKWRAP_FILE_PATH = '../npm-shrinkwrap.json';` ?
Not that it really matters of course...